src/DevGov/Notification/Item/Left.jsx

```diff
@@ -0,0 +1,42 @@
+/* INCLUDE: "common.jsx" */
+function href(widgetName, linkProps) {
+  linkProps = { ...linkProps };
+
+  if (props.referral) {
+    linkProps.referral = props.referral;
+  }
+
+  const linkPropsQuery = Object.entries(linkProps)
+    .filter(([_key, nullable]) => (nullable ?? null) !== null)
+    .map(([key, value]) => `${key}=${value}`)
+    .join("&");
+
+  return `/#/${REPL_DEVHUB}/widget/devhub.page.${widgetName}${
+    linkPropsQuery ? "?" : ""
+  }${linkPropsQuery}`;
+}
+/* END_INCLUDE: "common.jsx" */
+
+if (!props.type) {
+  return "Loading ...";
+}
+
+const type = props.type.split("/")[1];
+return props.type ? (
+  <>
+    {type == "like"
+      ? "liked your"
+      : type == "reply"
+      ? "replied to your"
+      : type == "edit"
+      ? "edited your"
+      : type == "mention"
+      ? "mentioned you in their"
+      : "???"}{" "}
+    <a className="fw-bold text-muted" href={href("post", { id: props.post })}>
+      DevHub post
```

Comment:
Then this <a tag should have 

```
href={to={href({
    widgetSrc: "${REPL_DEVHUB}/widget/app",
    params: {
      page: "post",
      id: props.id
    },
  })}
```

Although according to the story, the issue here is that for blogs, we should be pointing to blog instead:

```
to={href({
    widgetSrc: "${REPL_DEVHUB}/widget/app",
    params: {
      page: "blog",
    },
  })}
  ```
  
But we don't have access to the post type at this moment...

Maybe when we create the notification (author posting the blog), we could pass the postType of where it's coming from?

Reply:
yeah we don't know when it's a blog or post notification